server/runtime/README.adoc

```diff
@@ -125,39 +123,20 @@ An example configuration file looks as follows:
       </endpoints>
    </server>
 </infinispan>
+----
 
-```
-
-# Logging
-
-Logging is handled by JBoss Logging's LogManager. This is configured through a `logging.properties` file in the 
-`server/configuration` directory. The following is an example:
-
-```
-loggers=org.jboss.logmanager
-
-# Root logger
-logger.level=INFO
-logger.handlers=CONSOLE
-
-logger.org.jboss.logmanager.useParentHandlers=true
-logger.org.jboss.logmanager.level=INFO
+== Logging Configuration
+You configure server logging with `logging.properties` in the
+`server/configuration` directory.
 
-handler.CONSOLE=org.jboss.logmanager.handlers.ConsoleHandler
-handler.CONSOLE.formatter=PATTERN
-handler.CONSOLE.properties=autoFlush,target
-handler.CONSOLE.autoFlush=true
-handler.CONSOLE.target=SYSTEM_OUT
+JBoss Log Manager is the root logger but you can configure additional loggers:
```

Comment:
This statement is not quite correct. JBoss Logging LogManager is the implementation which follows the JDK LogManager specs and property-file format. We should probably point to official JDK docs on how this is done.

Reply:
thanks @tristantarrant I've removed that. I feel like we should put the details about logging as comments within the logging.properties file rather than duplicating them here. there's another PR for that.